forte/data/data_store.py

```diff
@@ -892,27 +911,93 @@ def co_iterator_annotation_like(
             yield entry
 
     def get(
-        self, type_name: str, include_sub_type: bool = True
+        self,
+        type_name: str,
+        include_sub_type: bool = True,
+        range_annotation: Optional[List[int]] = None,
```

Comment:
why is this a `List` or not simply a `Tuple[int, int]`?

Reply:
Yes, I think I can use a tuple instead.